src/main/resources/patchConfig.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name = "Checker">
+    <property name="charset" value="UTF-8"/>
+
+    <!-- do not change severity to 'error', as that will hide errors caused by exceptions -->
+    <property name="severity" value="warning"/>
+
+    <!-- haltOnException is required for exception fixes and reporting of all exceptions -->
+    <property name="haltOnException" value="false"/>
+
+    <!-- BeforeExecutionFileFilters is required for sources of java9 -->
+    <module name="BeforeExecutionExclusionFileFilter">
+        <property name="fileNamePattern" value="module\-info\.java$" />
+    </module>
+
+    <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchFilter">
+        <property name="file" value="/Users/hgh/Documents/GitHub/patch-filters/DiffReport/patch.txt"/>
```

Comment:
Please change the path a to a patch file. Others don't have this folder.
Also you generate patch.txt files in current dir https://github.com/checkstyle/patch-filters/pull/17/files#diff-bddb2f25480aacedb24f2f1035abc85dR143, you should use the same here

Reply:
so is necessary to using org.dom4j or other library to dynamic add this path or we have some simple way to modify it?

Reply:
I guess `./DiffReport/patch.txt` will work. Can you please test it?

Reply:
I test it, and it is not ok, and I understand the reason is that this patchConfig is used by checkstyle-tester and not by this project itself, so absolute path is necessary